app/testOnly/views/FeatureSwitchView.scala.html

```diff
@@ -43,5 +43,7 @@ <h3 class="govuk-heading-m">Frontend features</h3>
             </div>
         </fieldset>
         <button class="govuk-button" type="submit">Update global settings</button>
+        <button class="govuk-button" type="submit" name="enableall" value="true">Enable All Features</button>
```

Comment:
Can you use the existing convention for names

e.g. "feature-switch.enable-all-switches"

Reply:
Also I'm not sure if the value is needed

Reply:
Done